mesonbuild/backend/vs2010backend.py

```diff
@@ -1093,9 +1092,8 @@ def gen_regenproj(self, project_name, ofname):
         ET.SubElement(midl, 'TypeLibraryName').text = '%(Filename).tlb'
         ET.SubElement(midl, 'InterfaceIdentifierFilename').text = '%(Filename)_i.c'
         ET.SubElement(midl, 'ProxyFileName').text = '%(Filename)_p.c'
-        regen_command = [sys.executable,
-                         self.environment.get_build_command(),
-                         '--internal',
+        regen_command = self.environment.get_build_command() + \
+                         ['--internal',
```

Comment:
__[flake8]__
* [E127] continuation line over-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->